openssl_ocsp_responder/ocsp_responder.py

```diff
@@ -149,14 +161,110 @@ def set_revoked_certificate(self, certificate_path, revocation_time=None):
         self.crl[serial_number]['status'] = 'R'
 
         if revocation_time is None:
-            revocation_time = datetime.datetime.utcnow()
+            revocation_time = datetime.utcnow()
```

Comment:
It doesn't really matter (as long as you don't mix them in the same context), but why do you sometimes use `now` and sometimes `utcnow`?

Reply:
CRL index file require UTC time zone for some reason, with a "Z" suffix which also stands for UTC